models/models.go

```diff
@@ -108,6 +108,7 @@ func init() {
 		new(LFSMetaObject),
 		new(TwoFactor),
 		new(GPGKey),
+		new(GPGKeyImport),
```

Comment:
Can you add a migration to add this struct to DB (I know the sync below auto does this, however sync runs after migrations and could run into problems)

Reply:
@techknowlogick To understand you want me to do. I just need to add a  migrations script only sync the new struct ?

Reply:
Looking at just the previous migration should be done.

Reply:
Yes, just a sync2. The most recent migration is an excellent example.

Reply:
Ah, I just refreshed and saw you pushed the change. Thanks :)